### ISSUE TITLE
add ordering by date in (all?) tables

### ISSUE THREAD

**Issue by elenimijalis** [TAMU-CPT/lims-frontend#6]
It would be useful to have an ordering by date for environmental samples, for example

**Comment by hexylena** [comment]
Agreed. We should try and add ordering by as many fields as make sense, on all tables.
